apps/avatax/src/pages/api/webhooks/order-confirmed.ts

```diff
@@ -47,6 +48,11 @@ export default wrapWithLoggerContext(
       const { saleorApiUrl, token } = authData;
       const webhookResponse = new WebhookResponse(res);
 
+      if (payload.version) {
+        Sentry.setTag(ObservabilityAttributes.SALEOR_VERSION, payload.version);
+        loggerContext.set(ObservabilityAttributes.SALEOR_VERSION, payload.version);
```

Comment:
In the future we can think about a middleware that will add these values to every handled automatically